docs/minio-client-complete-guide.md

```diff
@@ -142,6 +142,15 @@ mc config host add gcs  https://storage.googleapis.com BKIKJAA5BMMU2RHO6IBB V8f1
 
 NOTE: Google Cloud Storage only supports Legacy Signature Version 2, so you have to pick - S3v2
 
+### Specify AccessKey and SecretKey through environment variables
+```sh
+export MC_HOSTS_<alias> = https://<Access Key>:<Secret Key> @ <YOUR-S3-ENDPOINT>
```

Comment:
There should not be any space around the `@` symbol

Reply:
The same is true for the spaces around `=`. 
No space is allowed around `=` when exporting an environment variable.